rmc-docs/src/tutorial/rmc-first-steps/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+// ANCHOR: code
+fn estimate_size(x: u32) -> u32 {
+    if x < 256 {
+        if x < 128 {
+            return 1;
+        } else {
+            return 3;
+        }
+    } else if x < 1024 {
+        if x > 1022 {
+            panic!("Oh no, a failing corner case!");
+        } else {
+            return 5;
+        }
+    } else {
+        if x < 2048 {
+            return 7;
+        } else {
+            return 9;
+        }
+    }
+}
+// ANCHOR_END: code
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use proptest::prelude::*;
+
+    #[test]
+    fn it_works() {
+        assert_eq!(estimate_size(1024), 7);
+    }
+
+// ANCHOR: proptest
+    proptest! {
+        #![proptest_config(ProptestConfig::with_cases(10000))]
+        #[test]
+        fn doesnt_crash(x: u32) {
+            estimate_size(x);
+        }
+    }
+// ANCHOR_END: proptest
+}
+
+fn __nondet() -> u32 {
```

Comment:
This function might confuse people. Should we include `prelude.rs` instead to "hide" some of the confusion?

Reply:
I sort of, kind of, hide it already, at least in that this code doesn't actually appear in the documentation.

But yes, we need to find a good way to address this. Right now it looked like there's no good way to include the prelude! The tests we have, at any rate, have to directly refer to it by relative path.

Fixing this is probably a major developer experience issue. My thought is: fix that, then revise this sample.

I create https://github.com/model-checking/rmc/issues/477 to take this as a note.

Reply:
Sounds good.